src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve) => {
+  const logo = document.querySelector('.logo');
+
+  if (logo) {
+    logo.addEventListener('click', () => {
+      resolve();
+    });
+  }
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('error'));
+  }, 3000);
+});
```

Comment:
The promise is rejected after 3 seconds without any condition. It would be better to have a condition that determines whether to resolve or reject the promise. If this is for testing purposes, make sure to remove or modify it in the production code.